constellations/gemini-admin/src/home/home.js

```diff
@@ -1,205 +1,103 @@
 "use strict";
-require("./home.styl");
+require("./home.sass");
 import React from "react";
 import API from "../api.js";
+import { Link } from "react-router-dom";
 
 export class HomePage extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            programs: [],
-            classes: [],
-            locations: [],
-            achieves: [],
-            semesters: [],
-            sessions: [],
-            rows: [],
-            selection: "",
-            noUnpub: "",
-            switch: false,
-        };
-        this.onClickPrograms = this.onClickPrograms.bind(this);
-        this.onClickClasses = this.onClickClasses.bind(this);
-        this.onClickLocations = this.onClickLocations.bind(this);
-        this.onClickAchievements = this.onClickAchievements.bind(this);
-        this.onClickSemesters = this.onClickSemesters.bind(this);
-        this.onClickSessions = this.onClickSessions.bind(this);
-        this.onClickPage = this.onClickPage.bind(this);
-        this.onClickPublishAll = this.onClickPublishAll(this);
-    }
-
-    componentDidMount() {
-        API.get("api/unpublished").then((res) => {
-            const unpublishedList = res.data;
-            this.setState({
-                programs: unpublishedList.programs,
-                classes: unpublishedList.classes,
```

Comment:
Right now looks like you're displaying fake data which is fine for "looks" purposes. The next step is to hook up actual unpublished classes to be displayed. So you're going to need this part here.

in a componentDidMount(), call API.get("api/unpublished") which is a Promise. When you receive data, setState of `classes` to `unpublishedList.classes`. You'll only need unpublished classes here.

From there, now that you `classes` is a list of objects, you should be able to render every class as a row. Reach out to people if you have questions :)